interfaces/repo.go

```diff
@@ -609,6 +609,38 @@ func (r *Repository) securitySnippetsForSnap(snapName string, securitySystem Sec
 	return snippets, nil
 }
 
+// RecordInterfacesAffectingSnap records all the security changes of a given snap.
+func (r *Repository) RecordInterfacesAffectingSnap(snapName string, rec Recorder) error {
```

Comment:
This is probably nicer as:

`func (r *Repository) SnapSpecification(securitySystem SecuritySystem, snapName string) (Specification, error) {`